src/cryptonote_core/service_node_quorum_cop.h

```diff
@@ -46,42 +47,27 @@ namespace service_nodes
       uint16_t version_major, version_minor, version_patch;
   };
 
-  struct quorum_checkpointing
+  struct testing_quorum
   {
-    std::vector<crypto::public_key> quorum_nodes;
-    BEGIN_SERIALIZE()
-      FIELD(quorum_nodes)
-    END_SERIALIZE()
-  };
-
-  struct quorum_uptime_proof
-  {
-    std::vector<crypto::public_key> quorum_nodes;
-    std::vector<crypto::public_key> nodes_to_test;
+    std::vector<crypto::public_key> validators;
+    std::vector<crypto::public_key> workers;
 
     BEGIN_SERIALIZE()
-      FIELD(quorum_nodes)
-      FIELD(nodes_to_test)
+      FIELD(validators)
+      FIELD(workers)
```

Comment:
Validators is fine for terminology, but I'm not crazy about "workers"--it makes me think of thread workers rather than service nodes being tested.  Perhaps "providers" would be a better choice?  Or "agents" (to borrow from contract theory terminology; you could even go slightly further along that line and replace "validator" with "monitor").

Reply:
Not a fan of provider, but I could live with monitor/validator and agents. Anyone else want to drop their 2cents?

Reply:
I will update this in a subsequent PR, it's a lot of menial work to get it renamed.